### DISCUSSION TITLE
Why nuxt is not same with package.json

### DISCUSSION THREAD

**Discussion by solmanter** [nuxt/nuxt#24686] [Questions] [answered]
> ## How to downgrade nuxt in project or node_modules?

### When nuxt running shows `3.8.2`
<img width="558" alt="Screenshot 2023-12-11 at 16 49 25" src="https://github.com/nuxt/nuxt/assets/104755033/036c2a56-fa5b-4639-93e0-70ff4d2d9375">

### But my package.json is `3.2.3`
<img width="316" alt="Screenshot 2023-12-11 at 16 50 01" src="https://github.com/nuxt/nuxt/assets/104755033/8145c1e3-6b52-41f2-b6ea-a7f3cd135ac8">

**Comment by Mobbbb** [comment]
`^` means upgrade minor version.If your version is `^3.2.3`, it will upgrade to `^3.9.9` probably, but won't upgrade to `^4.x.x`.If you want specific version.You should use `3.2.3`.

**Reply by solmanter** [reply]
Thank you so much @Mobbbb. it works but this error got. did you see before?
bunch of `workbox` logging and `Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
` error, can you help me? Please! 
<img width="655" alt="Screenshot 2023-12-12 at 20 49 06" src="https://github.com/nuxt/nuxt/assets/104755033/eed5a29f-6c88-44c1-ada3-5a8ce69dd852">

**Reply by Mobbbb** [reply, marked as answer]
There are many reasons probably.Maybe you should check router of  `pages/` or `layout` folder.
Check for errors in path references.
